recipe/meta.yaml

```diff
@@ -14,16 +14,19 @@ source:
   sha256: {{ hash }}
 
 build:
-  number: 2
+# Also skip for ARM64 due to Maturin no support for now, and when ppc64le AND python<3.10
```

Comment:
> ARM64 due to Maturin no support for now,

`maturin` should build just fine on `osx-arm64`.

Reply:
Maturin did complain about "bad CPU type" when building for ARM64? Not sure why then?

Reply:
You also need to put `maturin` into build using the `# [build_platform != target_platform]` selector.

Reply:
Sorry @xhochy for falling off; picking this up again. I just added maturin to build and selector as well.